keyboards/idobao/id80/v3/keymaps/via/config.h

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 peepeetee (@peepeetee)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#if defined(RGB_MATRIX_ENABLE)
+    #define VIA_QMK_RGBLIGHT_ENABLE
+#endif
```

Comment:
Just a heads up, the develop branch has support for rgb matrix, and handles it a bit better.  That lands at the end of the month. 
```suggestion
```

Reply:
i see it - https://github.com/qmk/qmk_firmware/blob/e7931289918221081cbe2a7ea5df27a5d86324db/quantum/via.c#L38